### ISSUE TITLE
Compilation err when building DLstreamer 2023-release with OpenVINO 2023.2.0

### ISSUE THREAD

**Issue by qianlongding** [dlstreamer/dlstreamer#399]
Hi,
I am trying to build DLstreamer from source code with OpenVINO 2023.2.0, 

But it works well with OpenVINO 2023.0.2.

err log:
```log
#41 38.47 [ 48%] Built target runtime_feature_toggling
#41 38.48 [ 48%] Linking CXX shared library ../../../../../../intel64/Release/lib/libdlstreamer_openvino.so
#41 38.49 [ 48%] Building CXX object src/monolithic/inference_backend/image_inference/CMakeFiles/image_inference.dir/image_inference.cpp.o
#41 38.61 In file included from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_loader.h:8,
#41 38.61                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:10,
#41 38.61                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/openvino_image_inference.h:13,
#41 38.61                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/image_inference.cpp:8:
#41 38.61 /opt/intel/openvino/runtime/include/ie/inference_engine.hpp:17:10: warning: #warning ("The Inference Engine API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html") [-Wcpp]
#41 38.61    17 | #        warning("The Inference Engine API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html")
#41 38.61       |          ^~~~~~~
#41 38.61 [ 48%] Built target dlstreamer_openvino
#41 38.62 In file included from /opt/intel/openvino/runtime/include/ie/ie_icnn_network.hpp:30,
#41 38.62                  from /opt/intel/openvino/runtime/include/ie/cpp/ie_cnn_network.h:31,
#41 38.62                  from /opt/intel/openvino/runtime/include/ie/cpp/ie_executable_network.hpp:29,
#41 38.62                  from /opt/intel/openvino/runtime/include/ie/ie_core.hpp:28,
#41 38.62                  from /opt/intel/openvino/runtime/include/ie/inference_engine.hpp:22,
#41 38.62                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_loader.h:8,
#41 38.62                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:10,
#41 38.62                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/openvino_image_inference.h:13,
#41 38.62                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/image_inference.cpp:8:
#41 38.62 /opt/intel/openvino/runtime/include/ngraph/function.hpp:13:10: warning: #warning ("The nGraph API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html") [-Wcpp]
#41 38.62    13 | #        warning("The nGraph API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html")
#41 38.62       |          ^~~~~~~
#41 38.64 [ 48%] Building CXX object src/monolithic/inference_backend/image_inference/openvino/CMakeFiles/image_inference_openvino.dir/model_builder.cpp.o
#41 38.75 In file included from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_loader.h:8,
#41 38.75                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:10,
#41 38.75                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:7:
#41 38.75 /opt/intel/openvino/runtime/include/ie/inference_engine.hpp:17:10: warning: #warning ("The Inference Engine API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html") [-Wcpp]
#41 38.75    17 | #        warning("The Inference Engine API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html")
#41 38.75       |          ^~~~~~~
#41 38.77 In file included from /opt/intel/openvino/runtime/include/ie/ie_icnn_network.hpp:30,
#41 38.77                  from /opt/intel/openvino/runtime/include/ie/cpp/ie_cnn_network.h:31,
#41 38.77                  from /opt/intel/openvino/runtime/include/ie/cpp/ie_executable_network.hpp:29,
#41 38.77                  from /opt/intel/openvino/runtime/include/ie/ie_core.hpp:28,
#41 38.77                  from /opt/intel/openvino/runtime/include/ie/inference_engine.hpp:22,
#41 38.77                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_loader.h:8,
#41 38.77                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:10,
#41 38.77                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:7:
#41 38.77 /opt/intel/openvino/runtime/include/ngraph/function.hpp:13:10: warning: #warning ("The nGraph API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html") [-Wcpp]
#41 38.77    13 | #        warning("The nGraph API is deprecated and will be removed in the 2024.0 release. For instructions on transitioning to the new API, please refer to https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html")
#41 38.77       |          ^~~~~~~
#41 38.85 [ 49%] Linking CXX static library ../../../../intel64/Release/lib/libmeta_overlay.a
#41 38.88 [ 49%] Built target meta_overlay
#41 38.90 [ 49%] Building CXX object src/monolithic/inference_backend/image_inference/async_with_va_api/image_inference_async/CMakeFiles/image_inference_async.dir/image_inference_async.cpp.o
#41 39.29 [ 49%] Linking CXX static library ../../../../intel64/Release/lib/libmeta_smooth.a
#41 39.32 [ 49%] Built target meta_smooth
#41 39.34 [ 49%] Building CXX object src/monolithic/gst/elements/gvametapublish/CMakeFiles/gvametapublish.dir/base/common.cpp.o
#41 39.51 [ 49%] Building CXX object src/monolithic/gst/elements/gvametapublish/CMakeFiles/gvametapublish.dir/base/gvametapublishbase.cpp.o
#41 39.92 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp: In function 'InferenceEngine::ColorFormat {anonymous}::formatNameToIEColorFormat(const string&)':
#41 39.92 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:180:48: error: 'NV12' is not a member of 'InferenceEngine::ColorFormat'
#41 39.92   180 |         {"NV12", InferenceEngine::ColorFormat::NV12}, {"I420", InferenceEngine::ColorFormat::I420},
#41 39.92       |                                                ^~~~
#41 39.92 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:180:94: error: 'I420' is not a member of 'InferenceEngine::ColorFormat'
#41 39.92   180 |         {"NV12", InferenceEngine::ColorFormat::NV12}, {"I420", InferenceEngine::ColorFormat::I420},
#41 39.92       |                                                                                              ^~~~
#41 39.92 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:183:99: error: no matching function for call to 'std::map<std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat>::map(<brace-enclosed initializer list>)'
#41 39.92   183 |         {"RGBA", InferenceEngine::ColorFormat::RGBX}, {"BGRA", InferenceEngine::ColorFormat::BGRX}};
#41 39.92       |                                                                                                   ^
#41 39.92 In file included from /usr/include/c++/11/map:61,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/include/inference_backend/image_inference.h:10,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:8,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:7:
#41 39.92 /usr/include/c++/11/bits/stl_map.h:290:9: note: candidate: 'template<class _InputIterator> std::map<_Key, _Tp, _Compare, _Alloc>::map(_InputIterator, _InputIterator, const _Compare&, const allocator_type&) [with _InputIterator = _InputIterator; _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   290 |         map(_InputIterator __first, _InputIterator __last,
#41 39.92       |         ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:290:9: note:   template argument deduction/substitution failed:
#41 39.92 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:183:99: note:   candidate expects 4 arguments, 8 provided
#41 39.92   183 |         {"RGBA", InferenceEngine::ColorFormat::RGBX}, {"BGRA", InferenceEngine::ColorFormat::BGRX}};
#41 39.92       |                                                                                                   ^
#41 39.92 In file included from /usr/include/c++/11/map:61,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/include/inference_backend/image_inference.h:10,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:8,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:7:
#41 39.92 /usr/include/c++/11/bits/stl_map.h:273:9: note: candidate: 'template<class _InputIterator> std::map<_Key, _Tp, _Compare, _Alloc>::map(_InputIterator, _InputIterator) [with _InputIterator = _InputIterator; _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   273 |         map(_InputIterator __first, _InputIterator __last)
#41 39.92       |         ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:273:9: note:   template argument deduction/substitution failed:
#41 39.92 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:183:99: note:   candidate expects 2 arguments, 8 provided
#41 39.92   183 |         {"RGBA", InferenceEngine::ColorFormat::RGBX}, {"BGRA", InferenceEngine::ColorFormat::BGRX}};
#41 39.92       |                                                                                                   ^
#41 39.92 In file included from /usr/include/c++/11/map:61,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/include/inference_backend/image_inference.h:10,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:8,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:7:
#41 39.92 /usr/include/c++/11/bits/stl_map.h:256:9: note: candidate: 'template<class _InputIterator> std::map<_Key, _Tp, _Compare, _Alloc>::map(_InputIterator, _InputIterator, const allocator_type&) [with _InputIterator = _InputIterator; _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   256 |         map(_InputIterator __first, _InputIterator __last,
#41 39.92       |         ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:256:9: note:   template argument deduction/substitution failed:
#41 39.92 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:183:99: note:   candidate expects 3 arguments, 8 provided
#41 39.92   183 |         {"RGBA", InferenceEngine::ColorFormat::RGBX}, {"BGRA", InferenceEngine::ColorFormat::BGRX}};
#41 39.92       |                                                                                                   ^
#41 39.92 In file included from /usr/include/c++/11/map:61,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/include/inference_backend/image_inference.h:10,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.h:8,
#41 39.92                  from /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:7:
#41 39.92 /usr/include/c++/11/bits/stl_map.h:250:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(std::initializer_list<std::pair<const _Key, _Tp> >, const allocator_type&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >; std::map<_Key, _Tp, _Compare, _Alloc>::allocator_type = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   250 |       map(initializer_list<value_type> __l, const allocator_type& __a)
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:250:7: note:   candidate expects 2 arguments, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:244:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(std::map<_Key, _Tp, _Compare, _Alloc>&&, const allocator_type&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >; std::map<_Key, _Tp, _Compare, _Alloc>::allocator_type = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   244 |       map(map&& __m, const allocator_type& __a)
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:244:7: note:   candidate expects 2 arguments, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:240:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(const std::map<_Key, _Tp, _Compare, _Alloc>&, const allocator_type&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >; std::map<_Key, _Tp, _Compare, _Alloc>::allocator_type = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   240 |       map(const map& __m, const allocator_type& __a)
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:240:7: note:   candidate expects 2 arguments, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:236:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(const allocator_type&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >; std::map<_Key, _Tp, _Compare, _Alloc>::allocator_type = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   236 |       map(const allocator_type& __a)
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:236:7: note:   candidate expects 1 argument, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:228:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(std::initializer_list<std::pair<const _Key, _Tp> >, const _Compare&, const allocator_type&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >; std::map<_Key, _Tp, _Compare, _Alloc>::allocator_type = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   228 |       map(initializer_list<value_type> __l,
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:228:7: note:   candidate expects 3 arguments, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:215:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(std::map<_Key, _Tp, _Compare, _Alloc>&&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   215 |       map(map&&) = default;
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:215:7: note:   candidate expects 1 argument, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:207:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(const std::map<_Key, _Tp, _Compare, _Alloc>&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   207 |       map(const map&) = default;
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:207:7: note:   candidate expects 1 argument, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:194:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map(const _Compare&, const allocator_type&) [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >; std::map<_Key, _Tp, _Compare, _Alloc>::allocator_type = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   194 |       map(const _Compare& __comp,
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:194:7: note:   candidate expects 2 arguments, 8 provided
#41 39.92 /usr/include/c++/11/bits/stl_map.h:185:7: note: candidate: 'std::map<_Key, _Tp, _Compare, _Alloc>::map() [with _Key = std::__cxx11::basic_string<char>; _Tp = InferenceEngine::ColorFormat; _Compare = std::less<std::__cxx11::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, InferenceEngine::ColorFormat> >]'
#41 39.92   185 |       map() = default;
#41 39.92       |       ^~~
#41 39.92 /usr/include/c++/11/bits/stl_map.h:185:7: note:   candidate expects 0 arguments, 8 provided
#41 39.93 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp: In function 'std::unique_ptr<InferenceBackend::ImagePreprocessor> {anonymous}::createPreProcessor(InferenceEngine::InputInfo::Ptr, size_t, const std::map<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> >&)':
#41 39.93 /root/dlstreamer/src/monolithic/inference_backend/image_inference/openvino/model_builder.cpp:224:81: error: 'NV12' is not a member of 'InferenceEngine::ColorFormat'
#41 39.93   224 |             input->getPreProcess().setColorFormat(InferenceEngine::ColorFormat::NV12);
#41 39.93       |                                                                                 ^~~~
#41 40.43 [ 49%] Building CXX object src/gst/bins/model_proc/CMakeFiles/model_proc.dir/model_proc_provider.cpp.o
#41 40.76 make[2]: *** [src/monolithic/inference_backend/image_inference/openvino/CMakeFiles/image_inference_openvino.dir/build.make:76: src/monolithic/inference_backend/image_inference/openvino/CMakeFiles/image_inference_openvino.dir/model_builder.cpp.o] Error 1
#41 40.76 make[1]: *** [CMakeFiles/Makefile2:3890: src/monolithic/inference_backend/image_inference/openvino/CMakeFiles/image_inference_openvino.dir/all] Error 2
#41 40.76 make[1]: *** Waiting for unfinished jobs....
#41 40.77 [ 50%] Building CXX object src/monolithic/gst/elements/gvametapublish/CMakeFiles/gvametapublish.dir/file/gvametapublishfile.cpp.o
#41 40.85 [ 50%] Building CXX object src/monolithic/gst/elements/gvametapublish/CMakeFiles/gvametapublish.dir/gvametapublish.cpp.o
#41 41.14 [ 51%] Building CXX object src/monolithic/gst/elements/gvametapublish/CMakeFiles/gvametapublish.dir/register.cpp.o
#41 41.23 [ 51%] Building C object src/gst/elements/gvafpscounter/CMakeFiles/gvafpscounter.dir/gstgvafpscounter.c.o
#41 41.25 [ 51%] Building CXX object src/gst/elements/gvafpscounter/CMakeFiles/gvafpscounter.dir/named_pipe.cpp.o
#41 41.41 [ 52%] Building CXX object src/monolithic/inference_backend/image_inference/async_with_va_api/image_inference_async/CMakeFiles/image_inference_async.dir/thread_pool.cpp.o
#41 41.43 [ 53%] Linking CXX static library ../../../../intel64/Release/lib/libimage_inference.a
#41 41.45 [ 53%] Linking CXX shared library ../../../../../intel64/Release/lib/libgvametapublish.so
#41 41.46 [ 53%] Built target image_inference
#41 41.54 [ 53%] Built target gvametapublish
#41 41.57 [ 54%] Linking CXX static library ../../../../intel64/Release/lib/libgvafpscounter.a
#41 41.61 [ 54%] Built target gvafpscounter
#41 42.48 [ 54%] Linking CXX static library ../../../../../../intel64/Release/lib/libimage_inference_async.a
#41 42.53 [ 54%] Built target image_inference_async
#41 42.88 [ 55%] Linking CXX static library ../../../../intel64/Release/lib/libmeta_aggregate.a
#41 42.94 [ 55%] Built target meta_aggregate
#41 44.06 [ 56%] Linking CXX static library ../../../../intel64/Release/lib/libmodel_proc.a
#41 44.12 [ 56%] Built target model_proc
#41 46.06 [ 56%] Built target sycl_meta_overlay
#41 46.06 make: *** [Makefile:136: all] Error 2

executor failed running [/bin/bash -xo pipefail -c virtualenv -p /usr/bin/python3 .env3 --system-site-packages     && source .env3/bin/activate     && source /opt/intel/oneapi/compiler/latest/env/vars.sh     && source /opt/intel/openvino/setupvars.sh     && cmake -DCMAKE_INSTALL_PREFIX=/opt/intel/dlstreamer -DTREAT_WARNING_AS_ERROR=OFF -DENABLE_VAAPI=ON -DENABLE_PERF_CAL=ON ..     && env -u no_proxy make -j`nproc`     && make install]: exit code: 2
make[2]: *** [workload/NVR/CMakeFiles/build_edge-iot-nvr.dir/build.make:76: workload/NVR/CMakeFiles/build_edge-iot-nvr] Error 1
make[1]: *** [CMakeFiles/Makefile2:8459: workload/NVR/CMakeFiles/build_edge-iot-nvr.dir/all] Error 2
[Uploading dlstreamer_erro.txt…]()
```

**Comment by cc0708** [comment]
> The last DL-Streamer's release "Release 2023.0" says it was built with
> 
> > the 2023.0.0 version of the OpenVINO™ Toolkit
> > (from "https://github.com/dlstreamer/dlstreamer/releases/tag/2023.0-release")
> > There were no updates to DL-Streamer since then...
> 
> If you require the latest OpenVINO then you either need to wait for an DL-Streamer update - or try to patch DL-Streamer on your own (regarding color formats) for now...

Thanks for your comment! Could you please help provide this patch to us? If can't, how difficult do you think if we create this path by ourselves? And could you point out the potential files regarding color formats?

**Comment by qianlongding** [comment]
Yes, we need latest version of OpenVINO, because for renset50 and mobilenet-v2 we saw better inference throughput on latest OpenVINO vs previous release. we need that improvement.

**Comment by qianlongding** [comment]
pls see the perf improvement for resnet50 and mobilenet-v2 on OpenVINO 23.2.
![resnet](https://github.com/dlstreamer/dlstreamer/assets/45531579/03034e42-dd13-4fd2-9780-2290de6fa9e7)

**Comment by qianlongding** [comment]
It is first case, we use the iGPU for video decoding(the format of decoded frame is NV12) and then perform inference on iGPU as well. Looks need to add NV12 and/or I420 (API1.0) back in OpenVINO 23.2, or Let DLstreamer to call new API2.0 of OpenVINO.

**Comment by pmalatyn** [comment]
we just released 2024.0 version with support of API 2.0. I highly recommend to check it.
